Tests/MongoSwiftSyncTests/SpecTestRunner/TestOperations/CollectionOperations.swift

```diff
@@ -234,10 +249,34 @@ struct DeleteOne: TestOperation {
     }
 }
 
-struct InsertOne: TestOperation {
-    let session: String?
+struct InsertOne: TestOperation, UnifiedOperationProtocol {
+    /// Document to insert.
     let document: BSONDocument
 
+    /// Optional identifier for a session entity to use.
+    let session: String?
+
+    /// Options to use while executing the operation.
+    let options: InsertOneOptions
+
+    private enum CodingKeys: String, CodingKey, CaseIterable {
+        case document, session
+    }
+
+    static var knownArguments: Set<String> {
+        Set(
+            CodingKeys.allCases.map { $0.rawValue } +
+                Mirror(reflecting: InsertOneOptions()).children.map { $0.label! }
```

Comment:
in practice I haven't found any cases where the label is nil, so I'm not sure why this is optional 🤔